pkg/lca/imagebasedupgrade.go

```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithOadpContent adds oadpContent to be used by the imagebasedupgrade.
+func (builder *ImageBasedUpgradeBuilder) WithOadpContent(
+	oadpConentName string, oadpConentNamespace string) *ImageBasedUpgradeBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Appending oadpContent name %s in namespace %s to the imagebasedupgrade",
+		oadpConentName, oadpConentNamespace)
+
+	builder.Definition.Spec.OADPContent = append(builder.Definition.Spec.OADPContent,
+		lcav1alpha1.ConfigMapRef{Name: oadpConentName, Namespace: oadpConentNamespace})
```

Comment:
```suggestion
	oadpContentName string, oadpContentNamespace string) *ImageBasedUpgradeBuilder {
	if valid, _ := builder.validate(); !valid {
		return builder
	}

	glog.V(100).Infof("Appending oadpContent name %s in namespace %s to the imagebasedupgrade",
		oadpContentName, oadpContentNamespace)

	builder.Definition.Spec.OADPContent = append(builder.Definition.Spec.OADPContent,
		lcav1alpha1.ConfigMapRef{Name: oadpContentName, Namespace: oadpContentNamespace})
```

Reply:
Did you suggest something new? Seems the same.